src/utility/util.js

```diff
@@ -0,0 +1,33 @@
+const { text } = require("express");
+const fs = require("fs");
+const path = require("path");
+
+const getNotesList = () => {
+  // get manga data fom the db file by reading, then parse the string and re.json
+
+  const notesJSON = fs.readFileSync(
+    path.join(__dirname, "../../db/db.json"),
+    "utf-8"
+  );
+
+  const notes = JSON.parse(notesJSON);
+
+  return notes;
+};
+
+const identifyNoteId = (notesDbArray, id) => {
```

Comment:
You haven't used this function anywhere. Try to delete any unnecessary code to avoid polluting your codebase.